app/Config/core.php

```diff
@@ -74,18 +75,18 @@
  *
  * @see ErrorHandler for more information on exception handling and configuration.
  */
-	Configure::write('Exception', array(
-		'handler' => 'ErrorHandler::handleException',
-		'renderer' => 'ExceptionRenderer',
-		'log' => true
-	));
+    Configure::write('Exception', array(
+        'handler' => 'ErrorHandler::handleException',
+        'renderer' => 'ExceptionRenderer',
+        'log' => true,
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 35 spaces but found 8